tests/storage/restore.go

```diff
@@ -882,19 +1092,22 @@ var _ = SIGDescribe("[Serial]VirtualMachineRestore Tests", func() {
 				deleteRestore(restore)
 			})
 
-			It("[test_id:6053]should restore a vm from an online snapshot", func() {
+			DescribeTable("should restore a vm from an online snapshot", func(restoreToNewVM bool) {
 				vm, vmi = createAndStartVM(tests.NewRandomVMWithDataVolumeAndUserDataInStorageClass(
 					cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros),
 					util.NamespaceTestDefault,
 					tests.BashHelloScript,
 					snapshotStorageClass,
 				))
 
-				doRestore("", console.LoginToCirros, true, 1)
+				doRestore("", console.LoginToCirros, true, 1, getTargetVMName(restoreToNewVM))
 
-			})
+			},
+				Entry("[test_id:6053] to the same VM", false),
+				Entry("to a new VM", true),
```

Comment:
This is a question, do we want to duplicate all the test and run them all with a restore to a new VM? @mhenriks what do you think?

Reply:
I did not re-used all of the tests, but I did most of them.
Well, the different strategies are very relevant for restoring a new VM IMO. I think in this case the overhead justifies the testing value.

Do you agree?
If not - which tests in your opinion are valuable to re-use? and why?

Thanks